mkdocs.yml

```diff
@@ -32,13 +35,10 @@ nav:
     - 'cp4d_hack_worker_nodes': roles/cp4d_hack_worker_nodes.md
     - 'cp4d_install': roles/cp4d_install.md
     - 'cp4d_install_services': roles/cp4d_install_services.md
-    - 'cp4d_wds': roles/cp4d_wds.md

```

Comment:
looks like a number of roles have been removed from this file for some reason?

Reply:
I just updated it, thanks